pkg/types/v2/file_existence.go

```diff
@@ -30,12 +30,13 @@ import (
 var defaultOwnership = -1
 
 type FileExistenceTest struct {
-	Name        string `yaml:"name"`        // name of test
-	Path        string `yaml:"path"`        // file to check existence of
-	ShouldExist bool   `yaml:"shouldExist"` // whether or not the file should exist
-	Permissions string `yaml:"permissions"` // expected Unix permission string of the file, e.g. drwxrwxrwx
-	Uid         int    `yaml:"uid"`         // ID of the owner of the file
-	Gid         int    `yaml:"gid"`         // ID of the group of the file
+	Name           string `yaml:"name"`           // name of test
+	Path           string `yaml:"path"`           // file to check existence of
+	ShouldExist    bool   `yaml:"shouldExist"`    // whether or not the file should exist
+	Permissions    string `yaml:"permissions"`    // expected Unix permission string of the file, e.g. drwxrwxrwx
+	Uid            int    `yaml:"uid"`            // ID of the owner of the file
+	Gid            int    `yaml:"gid"`            // ID of the group of the file
+	IsExecutableBy string `yaml:"isExecutableBy"` // whether file should be executable
```

Comment:
nit: can you change this comment to "name of group that file should be executable by"